backend/src/db/migrate/store.js

```diff
@@ -45,7 +69,7 @@ const createDefaultAdminUser = async (session) => {
     })
     try {
       await createAdminTxResultPromise
-      console.log('Successfully created default admin user') // eslint-disable-line no-console
+      console.log('Successfully created default admin user!') // eslint-disable-line no-console
```

Comment:
I allowed my self to put the exclamation mark behind the logging of created admin … as well and push …